frontend/lib/identification/card_detail_view/verification_code_view.dart

```diff
@@ -61,9 +61,12 @@ class _VerificationCodeViewState extends State<VerificationCodeView> {
                   children: [
                     Padding(
                       padding: EdgeInsets.all(padding),
-                      child: QrImage(
-                        data: const QrCodeUtils().createDynamicVerificationQrCodeData(activationCode, otpCode.code),
-                        version: QrVersions.auto,
+                      child: qr.QrImage.withQr(
+                        qr: qr.QrCode.fromUint8List(
+                          data: const QrCodeUtils().createDynamicVerificationQrCodeData(activationCode, otpCode.code),
+                          errorCorrectLevel: qr.QrErrorCorrectLevel.L,
```

Comment:
Don't we want to use Error Correction M here as well? Why not use auto ?

Reply:
Seems like there is no auto with binary QR codes: https://pub.dev/documentation/qr/2.1.0/qr/QrErrorCorrectLevel-class.html